webviz_subsurface/ert_jobs/well_connection_status.py

```diff
@@ -117,6 +117,10 @@ def main() -> None:
     args = parser.parse_args()
 
     df = _extract_well_connection_status(args.input)
+
+    # Create the output folder if it doen't exist
+    args.output.mkdir(parents=True, exist_ok=True)
```

Comment:
This will maybe also create a folder called `well_connection_status.parquet`?

Reply:
I will check.

Reply:
yes it does